std/node/os.ts

```diff
@@ -190,9 +190,9 @@ export function totalmem(): number {
   notImplemented(SEE_GITHUB_ISSUE);
 }
 
-/** Not yet implemented */
+/** Returns the operating system name as returned by uname(3) */
```

Comment:
<dl><dt>Nitpick:</dt><dd>If this function returns the host's <code>uname(3)</code>, then it might be better to call it <code>osName()</code> instead of <code>osType()</code>. The latter could be taken differently; i.e., referring to the operating system family (POSIX, Windows, Android, iOS, etc). At least, that was my immediate interpretation of it.</dd></dl>

Reply:
sure

Reply:
@AleksandrukTad changed them as you asked